From 8c0b46286faa55286f8f90ae2b0ef723d8992fc3 Mon Sep 17 00:00:00 2001
From: Daniel Jarai <jaraidaniel@gmail.com>
Date: Sat, 27 Jan 2018 20:56:36 +0100
Subject: [PATCH 17/17] board: Use the standard ION driver by default

Our common kernel finally has DMA-buffered ION, so let's use the
standard libion instead of TI's.

Change-Id: Iaa348016b66edc12329175fbe472ad7411819565
---
 BoardConfigCommon.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index 784b8c1..630eed3 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -37,7 +37,7 @@ OMAP_ENHANCEMENT_MULTIGPU := true
 BOARD_USE_TI_ENHANCED_DOMX := true
 BOARD_USE_TI_DUCATI_H264_PROFILE := true
 BOARD_USE_TI_DOMX_LOW_SECURE_HEAP := true
-BOARD_USE_TI_LIBION ?= true
+BOARD_USE_TI_LIBION ?= false
 ENHANCED_DOMX := true
 TARGET_SPECIFIC_HEADER_PATH += $(OMAP4_NEXT_FOLDER)/domx/omx_core/inc
 BOARD_USE_TI_CUSTOM_DOMX := true
-- 
2.7.4

